Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.Methods.cs

```diff
@@ -166,9 +166,16 @@ private void RenderMarkdown()
         private void UnhookListeners()
         {
             // Clear any hyper link events if we have any
-            foreach (Hyperlink link in _listeningHyperlinks)
+            foreach (object link in _listeningHyperlinks)
             {
-                link.Click -= Hyperlink_Click;
+                if (link is Hyperlink)
```

Comment:
change the if statements to reduce casting
```
if(list is Hyperlink hyperlink)
{
    hyperlink.Click -= Hyperlink_Click;
}
else if(link in Image image)
{
    image.Tapped -= NewImagelink_Tapped;
}
```

Reply:
done